src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -350,6 +350,7 @@ struct SvgNode
         SvgImageNode image;
     } node;
     bool display;
+    bool objectBBox = false;
```

Comment:
I'd need that for `mask` too. Not sure about patterns and filters

Reply:
I think we need to create a structure for clippath because the clippathUnit you add now is not a common property of SvgNode.
for example
`struct SvgClipPathNode` ?

Reply:
and
We are using userSpace as member variable name related to `userSpaceOnUse | objectBoundingBox` attribute. (Gradient case)
So, I think `bool userSpace` is better.
what do you think?

Reply:
@JSUYA for the gradient `objectBoundingBox` is the default units value. so `userSpace == true` only on request.
For clip and mask, the default is `userSpaceOnUse`, so by the analogy, I decided to use a different name, so that `objectBBox == true` only on request.

do you think it's less clear? if so I can change it